src/Compilers/CSharp/Portable/Symbols/FieldSymbol.cs

```diff
@@ -387,7 +387,18 @@ public virtual bool IsTupleField
         /// <summary>
         /// Returns True when field symbol is not mapped directly to a field in the underlying tuple struct.
         /// </summary>
-        internal virtual bool IsVirtualTupleField
+        public virtual bool IsVirtualTupleField
+        {
+            get
+            {
+                return false;
+            }
+        }
+
+        /// <summary>
+        /// Returns true if this is a field representing a Default element like Item1, Item2...
+        /// </summary>
+        public virtual bool IsDefaultTupleElement
```

Comment:
Some Fields represent elements. (Alice, Bob, Item10) and some fields do not (Rest).
Tuple fields can be virtual (not mapped directly to a field of underlying VT).
Every tuple has a set of elements that are always present - Item1, Item2 ... ItemN. Those are Default ones.

I think the naming is appropriate, since "Virtual" is mostly about the field - is it real field or an alias?
And "Default" is about elements - is it an element defined by the user or something added by compiler?


---
In reply to: [78254531](https://github.com/dotnet/roslyn/pull/13713#discussion_r78254531) [](ancestors = 78254531)